docs/components/operate/userguide/operate-feedback-and-questions.md

```diff
@@ -1,9 +1,11 @@
 ---
 id: operate-feedback-and-questions
 title: Giving feedback and asking questions
+description: "Have questions or feedback about Operate? Contact us."
 ---
 
 Have questions or feedback about Operate?
 
 Contact us at [Camunda Cloud Contact](/contact).
 
+[//]:# (Is this still up to date or are there additional methods for contact currently?)
```

Comment:
Linking to the contact page makes sense, but we could redirect to the Contact page and delete this.